README.md

```diff
@@ -37,18 +37,104 @@ The latest version from the release branch 'master' is automatically deployed to
 
 $ `git clone https://github.com/ita-social-projects/ZeroWaste.git`
 
-## Setup
+## Local setup
+First of all you need RVM to setup project. For the operating system Windows the optimal solution is to use <a href="https://docs.microsoft.com/en-us/windows/wsl/">WSL</a>.
 
 $ `bin/setup`
-or 
+or
 $ `bundle install`
 
+<b>PostgreSQL</b>
+
+<a href="https://www.postgresql.org/download/">Install PostgreSQL</a> for your operating system or subsystem.
+You can familiarize yourself with <a href="https://www.postgresql.org/docs/">PostgreSQL documentation</a>.
+
+In your local machine in cloned project in config folder rename database.yml.sample to database.yml. Make sure that the user and password match the data in this file. Port may be changed.
+
+For further work, make sure that you have a user 'postgres' with superuser. If is no that one do next:
+$ `sudo -u user psql user`
+$ `CREATE USER postgres SUPERUSER;`
+$ `CREATE DATABASE postgres WITH OWNER postgres;`
+
+If you're having trouble authenticating, you may need to reset your password. You can <a href="https://stackoverflow.com/questions/55038942/fatal-password-authentication-failed-for-user-postgres-postgresql-11-with-pg">read</a> instruction how to do it.
+
+<b>pg gem</b>
+
+Under certain circumstances bundle can do not install pg.
+
+To install manually:
+$ `sudo apt-get install libpq-dev`
+then
+$ `gem install pg`
+
+<b>Database configure</b>
+
+For correct operation of the migration, you need to rename the migration file `20220123171144_create_versions.rb` so that it is processed first.
+
+To create the necessary databases and update them:
+
+$ `rake db:create`
+then
+$ `rake db:migrate`
+
+$ `rake db:reset` can resolve some errors connected with database.
+
+<b>Redis</b>
+
+You need Redis for correct work.
+<a href="https://redis.io/docs/getting-started/">Install Redis</a> for your operating system or subsystem. You can familiarize yourself with
+<a href="https://redis.io/docs//">Redis documentation</a>.
+
+Installation for Ubuntu:
+
+$ `curl -fsSL https://packages.redis.io/gpg | sudo gpg --dearmor -o /usr/share/keyrings/redis-archive-keyring.gpg`
+
+$ `echo "deb [signed-by=/usr/share/keyrings/redis-archive-keyring.gpg] https://packages.redis.io/deb $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/redis.list`
+
+$ `sudo apt-get update`
```

Comment:
please try to add ability to copy commands from option from the right from command

Reply:
I`m not sure that is possible